### PR TITLE
update with lastest upstream

### PR DESCRIPTION
**Pull request by cirospaciari**

### CODE REVIEW

File: capi/libuwebsockets.cpp [hunk 1]
```diff
@@ -39,7 +40,10 @@ extern "C"
             sco.passphrase = options.passphrase;
             sco.ssl_prefer_low_memory_usage = options.ssl_prefer_low_memory_usage;
             sco.ssl_ciphers = options.ssl_ciphers;
-            
+<<<<<<< HEAD
+=======
+
```
Comment (Jarred-Sumner) [inline capi/libuwebsockets.cpp:45]: merge conflict

File: examples/EchoServer.cpp [hunk 2]
```diff
@@ -12,17 +12,12 @@ int main() {
...
         /* There are example certificates in uWebSockets.js repo */
-<<<<<<< HEAD
 	    // .key_file_name = "../misc/key.pem",
 	    // .cert_file_name = "../misc/cert.pem",
 	    // .passphrase = "1234"
-=======
-	    .key_file_name = "misc/key.pem",
-	    .cert_file_name = "misc/cert.pem",
-	    .passphrase = "1234"
->>>>>>> upstream/master
```
Comment (Jarred-Sumner) [inline examples/EchoServer.cpp:25]: lol

**Comment by Jarred-Sumner** [comment]
looks good just the merge conflict

File: examples/EchoServer.cpp (hunk 2 above)
Comment (cirospaciari) [reply examples/EchoServer.cpp:25]: lol will solve that

**Comment by Jarred-Sumner** [comment]
thank you

**Pull request merged**